Description: Disable coverage tests
 Coverage tests fail on Ubuntu Zesty, so they are disabled.
Author: Mathias Ertl <mati@jabber.at>
Bug: https://github.com/processone/fast_xml/issues/21
Last-Update: 2017-09-30
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
Index: erlang-p1-utils/rebar.config
===================================================================
--- erlang-p1-utils.orig/rebar.config
+++ erlang-p1-utils/rebar.config
@@ -4,7 +4,7 @@
     {platform_define, "^(15|16|17)", 'NEED_TIME_FALLBACKS'}
 ]}.
 
-{cover_enabled, true}.
+{cover_enabled, false}.
 {cover_export_enabled, true}.
 
 {xref_checks, [undefined_function_calls, undefined_functions, deprecated_function_calls, deprecated_functions]}.
